src/joystick_interrupt.cpp

```diff
@@ -113,6 +139,15 @@ class JoystickInterrupt
     }
     pub_int_.publish(status);
   };
+  static bool isEnabled(const int index)
+  {
+    return index >= 0;
```

Comment:
This also can be embedded in the if statements.
(I think the suitable location for this function is not in this class since the full name of this function is `JoystickInterrupt::isEnabled`)